### DISCUSSION TITLE
How can I upgrade asreview to its latest version?

### DISCUSSION THREAD

**Discussion by bagheria** [asreview/asreview#526] [Q&A] [answered]
How can I upgrade asreview to its latest version? What are the exact commands?

**Comment by J535D165** [comment, marked as answer]
See https://asreview.readthedocs.io/en/latest/intro/installation.html#upgrade-asreview

```
pip install --upgrade asreview
```

**Reply by J535D165** [reply]
To check the version of your current ASReview installation, run `asreview --version`. If this differs from https://pypi.org/project/asreview, then there is a newer version available.
